cli/k8s_client/k8s_client.go

```diff
@@ -59,6 +59,8 @@ const (
 
 	// CRD Finalizer name
 	TridentFinalizer = "trident.netapp.io"
+
+	CloudProviderAzure = "Azure"
```

Comment:
I suppose it's too late after running `tridentctl install`, since we need the host volume mount at install time to get access to azure.json.  But the operator runs in the cluster and could make that determination automatically, right?

https://learn.microsoft.com/en-us/azure/virtual-machines/instance-metadata-service?tabs=linux

Reply:
Yes, it is possible for operator to know which cloud provider is being used automatically. However, I think setting it manually in helm chart is easy to use and it doesn't rely on any external service. We can also add similar option in `tridentctl install` in the future.

Reply:
Another question...can this be tested in an Azure VM that isn't part of an AKS cluster?

Reply:
Nope, we rely on "/etc/kubernetes/azure.json" on the Node, which is generated by AKS component, Azure VM doesn't exist this file.